include/rnp/rnp.h

```diff
@@ -2269,6 +2269,22 @@ rnp_result_t rnp_identifier_iterator_next(rnp_identifier_iterator_t it,
  */
 rnp_result_t rnp_identifier_iterator_destroy(rnp_identifier_iterator_t it);
 
+/** Read from input and write to output
+ *
+ *  @param input stream to read data from
+ *  @param output stream to write data to
+ *  @return RNP_SUCCESS on success, or any other value on error
+ */
+rnp_result_t rnp_pipe(rnp_input_t input, rnp_output_t output);
```

Comment:
It could be better understood by user if function name is more descriptive and sticked to `rnp_output_` prefix, like `rnp_output_pipe()`.